### ISSUE TITLE
add a taxonomy argument to build

### ISSUE THREAD

**Issue by kevinVervier** [kevinVervier/metagm#10]
It would be nice to have an argument allowing users to pass their own taxonomy folder.
It is especially relevant as new versions will be created.

**Comment by MSMortensen** [comment]
It would also be nice to have an argument allowing users to set a prefix (or postfix) for the jobs. 
In general:
Request: An option to add a prefix (or postfix) to job names.
Reasoning: As log file names are derived from the job names it is currently impossible to run the same script twice in the same folder. A common use is to build two databases to compare how well they can classify the samples and then compare how well they perform. As the log files would be named identically this cannot be done in one folder at the moment. This is specifically a problem when running jobs in parallel, where logfiles cannot be moved. Another option would be to have an argument to specify a folder in which to store the log files. The most elegant solution might be to just output log files to a folder in output (output/logs/...).

Best,
Martin
